content/docs/tour/_index.md

```diff
@@ -1,34 +1,3 @@
 ---
-title: A Tour of Pulumi
```

Comment:
I wonder if we should redirect  this to "/docs/quickstart" or "/docs/get-started" if we're changing it to that? Based on my review, the tour seems to have been the precursor of "Get Started" and "Architecture & Concepts".

Reply:
I went back and forth on this and ended up picking `/docs/reference/concepts/` because that's where various other pages in the tour were largely going. But I could see linking to the Get Started page instead. I don't have a strong pref either way.

Reply:
BTW, if you think it'd be better to link to Get Started, I'd be happy to make the change. Let me know.

Reply:
Perhaps that is best? Since the "tour" was mostly a walkthrough with references to the concept docs.

Reply:
https://github.com/pulumi/docs/pull/1465